plugins/modules/remote_management/redfish/redfish_config.py

```diff
@@ -91,6 +91,16 @@
       - setting dict of EthernetInterface on OOB controller
     type: dict
     version_added: '0.2.0'
+  strip_etag_quotes:
+    required: false
```

Comment:
Can you apply the same changes from the other module here as well?

Reply:
Fixed. Apologies I missed that.